Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Data.Visualisation/Scott Plot/Plot/Plot.Add.cs

```diff
@@ -160,14 +160,20 @@ public BarPlot AddBar(double[] values, double[] errors, double[] positions, Colo
         public BarPlot[] AddBarGroups(string[] groupLabels, string[] seriesLabels, double[][] ys, double[][] yErr)
         {
             if (groupLabels is null || seriesLabels is null || ys is null)
+            {
                 throw new ArgumentException("labels and ys cannot be null");
+            }
 
             if (seriesLabels.Length != ys.Length)
+            {
                 throw new ArgumentException("groupLabels and ys must be the same length");
+            }
 
             foreach (double[] subArray in ys)
```

Comment:
brackets for foreach's as well please (Should be settable in a ReSharper style)

Reply:
Done